app/src/jobs/services/YoutubeMetadataService.js

```diff
@@ -40,13 +43,13 @@ function YoutubeMetadataService($q, $http) {
                     return getCurrentPlaylist(job).then(function (playlist) {
                         console.log('found playlist', playlist);
                         oldPlaylist.displayImages.thumbnails = [playlist.snippet.thumbnails.high];
-                        //oldPlaylist.playlistLength = playlist.contentDetails.itemCount + " vidéos";
-                        //oldPlaylist.accessibilityStr = oldPlaylist.playlistLength + ', ' + playlist.snippet.title;
+                        // oldPlaylist.playlistLength = playlist.contentDetails.itemCount + " vidéos";
+                        // oldPlaylist.accessibilityStr = oldPlaylist.playlistLength + ', ' + playlist.snippet.title;
                         oldPlaylist.title = playlist.snippet.title;
                         oldPlaylist.playlistId = playlist.id;
                         // TODO find next episode id maybe ?
                         oldPlaylist.videoId = videoId;
-                        oldPlaylist.targetUrl = encodeURIComponent('https://www.youtube.com/watch?v=' + videoId + '&list=' + playlist.id);
+                        oldPlaylist.targetUrl = 'https://www.youtube.com/watch?v=' + videoId + '&list=' + playlist.id;
```

Comment:
So, using `encodeURIComponent` would encode the `://` as well, which doesn't seem to be the case when I inspect the Network tab while looking at endscreen directly in Youtube. I don't think it's a breaker but it's closer to Youtube that way I think.